metron-deployment/vagrant/full-dev-platform/README.md

```diff
@@ -24,35 +26,38 @@ Any platform that supports these tools is suitable, but the following instructio
 
 1. Install Homebrew by following the instructions at [Homebrew](http://brew.sh/).
 
-2. Run the following command in a terminal to install all of the required tools.
+1. Run the following command in a terminal to install all of the required tools.
 
-  ```  
-  brew cask install vagrant virtualbox java
-  brew install maven git
-  ```
+    ```  
+    brew cask install vagrant virtualbox java docker
+    brew install maven git
+    ```
 
-3. Install Ansible by following the instructions [here](http://docs.ansible.com/ansible/intro_installation.html#latest-releases-via-pip).
+1. Install Ansible by following the instructions [here](http://docs.ansible.com/ansible/intro_installation.html#latest-releases-via-pip).
 
 ### Deploy Metron
 
-1. Install Vagrant Hostmanager.
+1. Build Metron
 
-  ```
-  vagrant plugin install vagrant-hostmanager
-  ```
+    ```
+    cd incubator-metron
+    mvn clean install -DskipTests
+    ```
 
```

Comment:
Metron is built during the provisioning process. There is a defect that makes a single "mvn install" call per release number required. See discussion [here](https://github.com/apache/incubator-metron/pull/538). My hope is the defect will be removed shortly. In the meantime, what would you think of either omitting this or adding some language that makes it clear that the manual install only needs to be called once per release until 538 is resolved (i.e. if we bumped the rev from 0.4.0, install would need to be called again)? Otherwise, following these instructions will add around 13 minutes to each build for no gain.